src/main/haskell/matching-logic/test/Kore/MatchingLogic/ProofSystem/ProofAssistantTest.hs

```diff
@@ -2188,7 +2193,7 @@ addGoal formula (NewGoalId goalId) proof =
             (GoalId goalId)
             -- TODO(virgil) remove this patternKoreToPure Meta nonsense and generate
             -- MetaMLPatterns directly
-            (patternKoreToPure Meta formula)
+            =<< mlErrorWrap (patternKoreToPure Meta formula)
```

Comment:
I guess that's instead of `purePattern <- mlErrorWrap (patternKoreToPure Meta formula)` .... Oh well...

Reply:
I admit I kind of cheated here, also taking advantage of the fact it was on a final position...